extensions/timeular/src/useTodayReport.ts

```diff
@@ -23,18 +23,37 @@ export const useTodayReport = (tracking: Tracking | null, activity?: Activity) =
     if (reportIsLoading) {
       return;
     }
+    const ms = Date.now() - new Date(tracking?.startedAt + "Z").getTime();
 
     const markStart = Date.now();
 
+    const totalBookedTime = entries.reduce((acc, val) => {
+      if (!val.duration.stoppedAt || !val.duration.startedAt) {
+        // Skip entries that are still running (no stoppedAt)
+        return acc;
+      }
+
+      const [startedAt, stoppedAt] = [
+        new Date(val.duration.startedAt + "Z").getTime(),
+        new Date(val.duration.stoppedAt + "Z").getTime(),
+      ];
```

Comment:
Let's make it a tad simpler
```ts
const startedAt = new Date(val.duration.startedAt + "Z").getTime();
const stoppedAt = new Date(val.duration.stoppedAt + "Z").getTime();

```

Reply:
Yeah you're right. no need to get fancy haha. I'll split them out.